sycl/source/detail/platform_info.hpp

```diff
@@ -49,11 +49,27 @@ struct get_platform_info<std::vector<std::string>, info::platform::extensions> {
   }
 };
 
+template <> struct get_platform_info<bool, info::platform::ext_oneapi_P2P> {
+  static bool get(RT::PiPlatform plt, const plugin &Plugin) {
+
+    std::string vendor_name =
+        get_platform_info<string_class, info::platform::vendor>::get(plt,
+                                                                     Plugin);
+    bool result = (vendor_name == "NVIDIA Corporation") ? true : false;
+    return result;
```

Comment:
This behavior should be handled by the backends, i.e.
```suggestion
    return get_platform_info<bool, info::platform::ext_oneapi_P2P>::get(plt, Plugin);
```
and then implement cases for `PI_PLATFORM_INFO_ext_oneapi_P2P` in `piPlatformGetInfo`.